### ISSUE TITLE
Add HTTPS

### ISSUE THREAD

**Issue by sskhandek** [ReflectionsProjections/rp2017#13]

**Comment by sskhandek** [comment]
This is on hold since we don't have a deployment strategy yet

**Comment by pranaygp** [comment]
https://letsencrypt.org/

Don't know how relevant that is, but I've heard people talk about this before. It's free too!

**Comment by bluedrops** [comment]
@pranaygp : Yup, that's what I thought of too; unfortunately we can't do it for static pages though.

We're working on getting a VM from ACM to deploy on so we can migrate away from Github Pages. Once we do that, we can do letsencrypt 😃
